### PR TITLE
Refactoring env variables

### PR DESCRIPTION
**Pull request by mtrunkat**
BEFORE merge we have to first release new `apify-shared`, `apify` and update versions here!

But please check this pull request now..

### CODE REVIEW

**Comment by drobnikj** [comment]
@mtrunkat Tests failed on travis. Should I check it? Or are you still working on it?

**Comment by mtrunkat** [comment]
SDK and apify-shared is not published yet. It's going to work afterwards.

pá 7. 9. 2018 v 10:00 odesílatel Jakub Drobník <notifications@github.com>
napsal:

File: src/lib/files.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
 const loadJson = require('load-json-file');
 const writeJson = require('write-json-file');
 const fs = require('fs');
+const path = require('path');
 const rimraf = require('rimraf');
 const Promise = require('bluebird');
```
Comment (mnmkng) [inline src/lib/files.js]: We should get rid of this when using `async` `await`.

File: src/lib/utils.js [hunk 2]
```diff
@@ -92,20 +114,16 @@ const setLocalConfig = async (localConfig, actDir) => {
...
-    createFolderSync(path.join(keyValueStoresDir, DEFAULT_LOCAL_STORES_ID));
+    ensureFolderExistsSync(actDir, getLocalDatasetPath());
+    ensureFolderExistsSync(actDir, getLocalRequestQueuePath());
+    ensureFolderExistsSync(actDir, getLocalKeyValueStorePath());
 
     // Update gitignore
     const gitingore = path.join(actDir, '.gitignore');
     if (fs.existsSync(gitingore)) {
-        fs.writeFileSync(gitingore, `\n${DEFAULT_LOCAL_EMULATION_DIR}`, { flag: 'a' });
+        fs.writeFileSync(gitingore, `\n${LOCAL_ENV_VARS[DEFAULT_LOCAL_STORAGE_DIR]}`, { flag: 'a' });
```
Comment (drobnikj) [inline src/lib/utils.js]: It should be just `DEFAULT_LOCAL_STORAGE_DIR`, not `LOCAL_ENV_VARS[DEFAULT_LOCAL_STORAGE_DIR]`. Or maybe better `getLocalStorageDir()`

File: src/lib/utils.js [hunk 3]
```diff
@@ -92,20 +114,16 @@ const setLocalConfig = async (localConfig, actDir) => {
...
     } else {
-        fs.writeFileSync(gitingore, `${DEFAULT_LOCAL_EMULATION_DIR}\nnode_modules`, { flag: 'w' });
+        fs.writeFileSync(gitingore, `${LOCAL_ENV_VARS[DEFAULT_LOCAL_STORAGE_DIR]}\nnode_modules`, { flag: 'w' });
```
Comment (drobnikj) [inline src/lib/utils.js]: Same as line 124

**Pull request merged**